### PR TITLE
dont put members on sys message

### PR DESCRIPTION
**Pull request by mmaxim**

### CODE REVIEW

File: go/teams/chatmessaging.go [hunk 1]
```diff
@@ -49,16 +49,10 @@ func SendTeamChatWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, tea
 	}
 	username := g.Env.GetUsername()
 	subBody := chat1.NewMessageSystemWithAddedtoteam(chat1.MessageSystemAddedToTeam{
-		Adder:          username.String(),
-		Addee:          user,
-		Role:           role,
-		Team:           team,
-		Owners:         ownerNames,
```
Comment (joshblum) [inline go/teams/chatmessaging.go]: dont' need to get `TeamDetails` either, can delete a bunch of the above code. Why not remove this from the message struct?

File: shared/constants/chat2/message.tsx [hunk 2]
```diff
@@ -634,18 +633,15 @@ const uiMessageToSystemMessage = (
 ): Types.Message | null => {
   switch (body.systemType) {
     case RPCChatTypes.MessageSystemType.addedtoteam: {
-      // TODO @mikem admins is always empty?
-      const {adder = '', addee = '', team = '', admins = null} = body.addedtoteam || {}
+      const {adder = '', addee = '', team = ''} = body.addedtoteam || {}
       const roleEnum = body.addedtoteam ? body.addedtoteam.role : undefined
       const role = roleEnum ? TeamConstants.teamRoleByEnum[roleEnum] : 'none'
-      const isAdmin = (admins || []).includes(minimum.author)
```
Comment (joshblum) [inline shared/constants/chat2/message.tsx]: can't we just fetch the role of the author and keep this bit in?

Comment (mmaxim) [reply shared/constants/chat2/message.tsx]: It doesn't do anything important, just changes one word.

Comment (joshblum) [reply shared/constants/chat2/message.tsx]: but most of those system messages have the `manage members` part, why not keep it?

Comment (mmaxim) [reply shared/constants/chat2/message.tsx]: Just seemed like a waste of effort

Comment (mmaxim) [reply shared/constants/chat2/message.tsx]: Note that it didnt even work before, since it only looked in the admin list, and not the owners

**Pull request merged**